src/devices/vdml.c

```diff
@@ -30,17 +30,7 @@ extern void registry_init();
 extern void port_mutex_init();
 
 int32_t claim_port_try(uint8_t port, v5_device_e_t type) {
-	if (!VALIDATE_PORT_NO(port)) {
-		errno = ENXIO;
-		return PROS_ERR;
-	}
-	if (registry_validate_binding(port, type) != 0) {
-		return PROS_ERR;
-	}
-	if (!port_mutex_take(port)) {
-		errno = EACCES;
-		return PROS_ERR;
-	}
+	claim_port(port, type, 0);
```

Comment:
I know it may be redundant code, but it turns out the difference between `claim_port_try` and `claim_port` is that the latter saves a copy of the device (`v5_smart_device_s_t* device = registry_get_device(port)`) while the former does not. Compiling with `-Wunused-variable` (as we do) complains about this.